File: src/test/kotlin/com/rankweis/uppercut/karate/psi/KarateFormattingTest.java
package com.rankweis.uppercut.karate.psi;

import com.intellij.psi.formatter.FormatterTestCase;

public class KarateFormattingTest extends FormatterTestCase {

//    @BeforeEach
//    public void setUp() throws Exception {
//        super.setUp();
//    }

    public void testComplicated() throws Exception {
        doTest();
    }

    public void testRandom() throws Exception {
        doTest();
    }

    public void testJs() throws Exception {
        doTest();
    }

    @Override
    public String getBasePath() {
        return "testData";
    }

    @Override
    public String getTestDataPath() {
        return "src/test";
    }

    @Override
    public String getFileExtension() {
        return "feature";
    }
}

File: src/test/kotlin/com/rankweis/uppercut/karate/psi/KarateFormattingTest.kt
package com.rankweis.uppercut.karate.psi

import com.intellij.psi.formatter.FormatterTestCase

class KarateFormattingTest : FormatterTestCase() {

    fun testFormatter() {
        doTest()
    }

    override fun getBasePath(): String {
        return "testData"
    }

    override fun getTestDataPath(): String {
        return "src/test"
    }

    override fun getFileExtension(): String {
        return "feature"
    }
}